### ISSUE TITLE
Is the name of this package appropriate?

### ISSUE THREAD

**Issue by Datseris** [chakravala/Geophysics.jl#1]
Continuing a discussion in the announcement post https://discourse.julialang.org/t/ann-geophysics-jl/49097 :

> Isn’t the name of this package overly general and could even miss-lead a newcomer? I’d say Geophysics is a really large branch of the geosciences, but this package seems to focus specifically on providing values of pressure, temperature and other variables of the atmosphere at a given height. Wouldn’t a more fitting name be something like Atmospheres.jl or AtmosphericVariables.jl?

**Comment by chakravala** [comment]
I am interested in adding also features for oceans, and perhaps temperature inside the Earth also, as well as electrical and magnetic properties of planets. In this package, any simple Geophysical properties of planets may be added. However, I only immediately need the simple properties of the atmosphere, so that's where it is starting off for me. `README.md` is now updated:

Originally, I considered using a different name, but `Atmospheres` is too specific, once I add other things to it later on. Since I am not a geophysics expert, I cannot immediately provide a fast implementation of all the other simple geophysical data. My hope is that other people will contribute other *simple* geophysical data about other planets, its a collaborative effort.

Complicated models will be excluded from this package, as it is only intended to provide a **foundation** for geophysical data and constants of various planets, more complicated models should be built separately in packages that build on `Geophysics`. For example, some geographic conditions can be calculated externally, and then `Geophysics` is used to load that data.

**Comment by RafaelBGu** [comment]
Suggestion: Planetary_Geophysics.jl

**Comment by rafael-guerra-www** [comment]
From the package description would it be more appropriate to call it ```PlanetExteriorGeophysics```?

**Comment by chakravala** [comment]
I prefer `Geophysics` since planet is a redundant term, it is too loong of a name you are suggesting, i prefer a simple name that doesnt include redundant extra titles.

If you want to register your own Geophysics package, that is still possible since the new package manager supports naming multiple packages with the same name.

**Comment by rafael-guerra-www** [comment]
Ok. Here is another suggestion:  ```GeospherePhysics```, following the reference  [Williams, R. [2012] The Earth System, USGS](https://pubs.usgs.gov/pp/p1386a/pdf/notes/1-8hydrocycle_508.pdf)

**Comment by chakravala** [comment]
Thanks for the suggestion, but you have not yet identified any problem or issue with the current name. Any suggestions to change the name will be ignored, unless you at least specify a problem or issue with the current name.

**Comment by rafael-guerra-www** [comment]
One important chapter in Geophysics is the theory of seismic wave propagation in acoustic, elastic and anisotropic media. This package does not seem to be contemplating this, not allowing simulating  seismic waves in complex three-dimensional solid Earth models. Nor the inversion and mapping of subsurface properties from geophysical measurements such as gravity, seismic, electrical, GPR and/or magneto-telluric surveys.

**Comment by chakravala** [comment]
You are mistaken, geophysical properties are allowed, hence the name. Geophysical data can be added to this package, including seismic or magnetic property data.

The goal of this package is to provide quickly accessible geophysical data to load in an instant and this should not require complex 3D models. My higher dimensional models are separate from this package, since they require additional dependencies (all of which is not necessary for quick and basic geophysics data).

This package is primarily for my personal use and for package designs I am developing. If it is not suitable for you, I encourage you to implement your own vision of a `Geophysics` package, and to stop worrying about my package unless you intend to use this package or contribute.

**Comment by rafael-guerra-www** [comment]
As I never started worrying, so no worries.
